spec/factories/hesa/students.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
```

Comment:
Again, hopefully we'll be discarding this class once we've cleared up the data. Definitely shouldn't be building too much stuff on it.